sabre.js

```diff
@@ -88,64 +85,84 @@ try {
         // NOTE: `solcSnapshot` has the same interface as `solc`
         const solcSnapshot = solc.setupMethods(requireFromString(solcString), 'soljson-' + releases[version] + '.js');
 
-        let compiledData;
+        /* Parse all the import sources and the `sourceList` */
 
-        try {
-            compiledData = compiler.getCompiledContracts(input, solcSnapshot, solidity_file_name);
-        } catch (e) {
-            console.log(chalk.red(e.message));
-            process.exit(1);
-        }
+        Profiler.resolveAllSources(resolver, [solidity_file_path], solcSnapshot)
+            .then(resolved => {
+                const sourceList = Object.keys(resolved);
+                console.log(sourceList);
```

Comment:
Is this a leftover debugging code?

Reply:
Yeah, I'll remove this. Thanks for pointing it out 👍